### ISSUE TITLE
request for more focus on ps Vita, as it is a platform that requires more attention

### ISSUE THREAD

**Issue by Sal-gamer** [Hydr8gon/NooDS#91]
PS Vita is a platform with its own tools, resources and means of work, it requires more knowledge, time and optimization than other platforms, I would like to request a special dedication to it, please thank you in advance.

**Comment by nadiaholmquist** [comment]
As far as I know the PSVita port was started as more of a curiosity than anything, the emulator wasn't made with it in mind in the beginning and as a result you're probably setting yourself up for disappointment if you're expecting it to ever run full speed.

**Comment by Sal-gamer** [comment]
from the beginning he said that he was testing and that he did not believe that the Nintendo DS would run 100% on PS Vita, and ok!  but I'm asking for more attention and dedication on ps Vita, since it's the platform that most needs optimization and attention, I just want to see what this will give if more attention is paid to Vita, the DeSmuME emulator currently runs better than the NooDS emulator, this means that it is currently less optimized than the 2016 port of DeSmuME, and it looks like it had 1 week of development and release was from August 2nd to August 9th, 2016, until they abandoned, 1 week they did a lot there in 2016 , and 2022 the NooDS emulator runs worse than a week of DeSmuME development, if an older emulator runs better than a new one, and because there are things to improve and that's why I'm asking for more focus, attention, optimization and dedication for PS Vita (I'm not belittling the work of NooDS, I'm just saying that it can improve and that DeSmuME runs better, NooDS is very good)

**Comment by nadiaholmquist** [comment]
I think DeSmuME had an ARMv7 JIT already, that would explain the better performance.

**Comment by Sal-gamer** [comment]
anyway, I just wanted more focus and commitment to make it minimally playable, I think that with many platforms and support for them, it leaves the Vita a little aside, as it needs some attention 😞

**Comment by Sal-gamer** [comment]
Rinnegatamente was working on the port of MelonDS for PS Vita but said that he was waiting for some things to come out to the emulator, then bring it to PS Vita, and there is possibly a person trying to port DraStic from Android to PS Vita, this is the third DS emulator for ps Vita and everyone has an initial release but soon after they just abandon it, I wanted noods to be different from the other 2 abandoned emulators for ps Vita, about DraStic if I'm not mistaken, I'm carrying it using metodo do the flow to run, using VitaGL and possibly other tools like sharckfood, Glsl and Opensles, I don't know if it uses , I'm just saying that you can use other resources besides VitaGL to port DraStic, you can use the fourth core/thread of the processor vita , you can redo some things, I don't know, I just know that there's an image of it already in vita , but even if DraStic is ported I would like NooDS to cintunuse , I wanted to ber on what can happen with the emulator, having more focus  and commitment to ps Vita

**Comment by Sal-gamer** [comment]
![Uploading legend.png…]()

**Comment by Sal-gamer** [comment]
![Uploading legend.png…]()

**Comment by nadiaholmquist** [comment]
If someone is working on a port of DraStic that is the one you should be looking out for. DraStic is optimized to run on lower power mobile devices, sacrificing accuracy where possible, and may have a chance to run well on the Vita.

Other DS emulators like NooDS were built with the PC and thus more powerful hardware in mind, and would require a lot of optimizations and hacks that likely don't make sense on higher power devices to run well on something with a quite slow CPU like the Vita.

**Comment by Sal-gamer** [comment]
as I said, I hope that NooDS gets optimized and that they pay more attention to PS Vita, I know that NooDS is more optimized for PC and devices that use a lot of hardware and do little, so I'm asking for more attention and focus on ps Vita since the others have several emulators as you said and are poorly optimized because the hardware uses a lot and to have a satisfactory result, and is not optimized

**Comment by Sal-gamer** [comment]
![legend](https://user-images.githubusercontent.com/55700359/153995813-ad4f6ca1-5010-474c-a0aa-cdf02cf27461.png)
Imagen DraStic on vita

**Comment by Sal-gamer** [comment]
the developer who posted this image, and extremely skilled and very dear in the PS Vita scene, fuck dude, this image is official, makes me very excited, still, I want to see what DraStic, made practically from scratch, can be able to do in ps Vita

**Comment by waifro** [comment]
im not sure if you understood the complications behind developing an emulator for another hardware. Theres more then just "appreciation" of the vita console. Ngl i never worked with this console in my life, but from my litte understanding, its too hacky to develop for, even for regular developers, thanks to Sony's overthinked hw<->sw engineering. Im probably mostly wrong, ~~but the psvita is classified to be left to die on its own.~~ There's little to none interest to the devs; lacks of community primarly.

**Comment by Sal-gamer** [comment]
> não tenho certeza se você entendeu as complicações por trás do desenvolvimento de um emulador para outro hardware. Há mais do que apenas "apreciação" do console vita. Ngl eu nunca trabalhei com este console na minha vida, mas pelo meu pequeno entendimento, é muito hacky para desenvolver, mesmo para desenvolvedores regulares, graças à engenharia hw<->sw da Sony. Provavelmente estou errado,~mas o psvita é classificado para ser deixado para morrer por conta própria.~Há pouco ou nenhum interesse para os desenvolvedores; falta de comunidade principalmente.

yes, there is a low number of developers of emulators for PS Vita, but it doesn't mean that it is zero , more and more developers arrive in the Vita scene, currently it is growing, I've been following it and more and more there are homebrew, ports and some doing emulator like mgaba among others making smaller emulators, but I believe there will still be more Vita emulator developers.  but community and what is not lacking for PS vita, there is community in Russian, Braseiro, Spanish, English and so on.

**Comment by EternalsDarkhole** [comment]
This can be close as a dup of https://github.com/Hydr8gon/NooDS/issues/64
As I read from that issue Vita is a slow platform for ds  emulator 🤷

**Comment by Sal-gamer** [comment]
> Isso pode ser quase uma duplicação de [# 64](https://github.com/Hydr8gon/NooDS/issues/64) Como eu li sobre esse problema, o Vita é uma plataforma lenta para emulador de ds🤷

if you say that ps vita is slow to run ds it's ok.....I'll just wait one day to come out draatic , and an armv6 / armv7 emulator made for armv7 with extremely low requirements, who knows one day they will use the lib method. just to run the emulator or Drastic's code is open, or some workaround, drastic even runs on a calculator, it's good, (note: it's not duplicated or almost, I just asked for more attention/focus for ps vita since other platforms have several ds emulators, and it doesn't even need much optimization, I believe that noods can be big and want a stable and functional version vita, I'm asking for more love in vita not asking because it doesn't run, I'm asking for dedication to ps vita.
